### ISSUE TITLE
Support Entity upsert by ids

### ISSUE THREAD

**Issue by douglarek** [milvus-io/milvus#4669]
**Please state your issue using the following template and, most importantly, in English.**

**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. E.g. I wish I could use Milvus to do [...]

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
Add any other context, code examples, or references to existing implementations about the feature request here.

**Comment by douglarek** [comment]
So far milvus does not support records upsert like relational database, you must delete them first then insert again, this is really not so good.

**Comment by sahuang** [comment]
Are you talking about entity update? You can refer to https://github.com/milvus-io/milvus/issues/3198 for any update.

**Comment by douglarek** [comment]
> Are you talking about entity update? You can refer to #3198 for any update.

thanks, maybe i am not making it so clear, `upsert` is an operation that inserts rows into a database table if they do not already exist, or updates them if they do.

**Comment by sahuang** [comment]
Yeah, I understand that. As long as Milvus supports entity update, upsert will be an easy job. This is on our to-do list but needs more considerations.

**Comment by jingkl** [comment]
For this question refer to the issue #4820

**Comment by yhmo** [comment]
According to the roadmap: https://wiki.lfaidata.foundation/display/MIL/Milvus+2.0+Roadmap+and+Time+schedule
The upsert() will available in v2.2.

**Comment by xiaofan-luan** [comment]
We should has this feature by 2.1 release

**Comment by xiaofan-luan** [comment]
upsert is in 2.3 and already on master branch

**Comment by cyrusvahidi** [comment]
> upsert is in 2.3 and already on master branch

thanks. is there a guide on how to update from previous versions with docker?

**Comment by xiaofan-luan** [comment]
There is the upgrade guide.
https://milvus.io/docs/upgrade_milvus_cluster-operator.md

Milvus 2.3 is not released yet and will be released mid June
